Vueify 8.4.1 wants babel-runtime v6

Hi

I'm receiving the following error when installing this package...

`
npm ERR! peerinvalid The package babel-runtime@5.8.38 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer vueify@8.4.1 wants babel-runtime@^6.0.0
`

Having a look at the package.json dependencies it says it wants 

`
"babel-runtime": "^5.8.34",
"vueify": "^8.3.0"
`

Is this just an issue with me?!

I deleted the node_modules directory and tried installing again but have this error...

`
npm WARN peerDependencies The peer dependency babel-runtime@^6.0.0 included from vueify will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "laravel-elixir-vueify"
npm ERR! node v4.4.3
npm ERR! npm  v2.15.1
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package babel-runtime@5.8.38 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer vueify@8.4.1 wants babel-runtime@^6.0.0
`

Any ideas how I can install it?

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "babel-core": "^6.3.26",
     "babel-plugin-transform-runtime": "^6.3.13",
     "babel-preset-es2015": "^6.3.13",
-    "babel-runtime": "^5.8.34",
+    "babel-runtime": "^6.0.0",
     "vue": "^1.0.20",
     "vueify": "^8.3.0",
     "vue-hot-reload-api": "^1.2.2",

